helmclient.go

```diff
@@ -382,10 +388,12 @@ func (c *Client) InstallFromTarball(path, ns string, options ...helmclient.Insta
 		defer c.closeTunnel(t)
 
 		release, err := c.newHelmClientFromTunnel(t).InstallRelease(path, ns, options...)
-		if IsReleaseNotFound(err) {
-			return backoff.Permanent(releaseNotFoundError)
-		} else if IsCannotReuseRelease(err) {
+		if IsCannotReuseRelease(err) {
 			return backoff.Permanent(cannotReuseReleaseError)
+		} else if IsReleaseNotFound(err) {
+			return backoff.Permanent(releaseNotFoundError)
```

Comment:
Any reason why some errors are masked and some aren't?

Reply:
This is just moved down so the order is alphabetical. I didn't change anything so don't ask me for reasons :)

Reply:
PTALA

Reply:
So why did you change all the errors now? Feels like this will now just break left and right. Are you confident with this? Because, I am not.

Reply:
What can be broken? All errors are from the same package.